lib/plugins/inspec-reporter-junit/README.md

```diff
@@ -1,19 +1,17 @@
 # junit and junit2 reporters
 
-This is the implementation of the junit and junit2 XML reporter.
+This is the implementation of the junit and junit2 XML reporters.
 
-## To Install This Plugin
+## To Install These Plugins
 
-This plugin is included with inspec. There is no need to install it separately.
+This plugin is included with InSpec. There is no need to install it separately.
 
-## What This Plugin Does
+## What These Plugins Do
 
-This reporter generates an XML report in Apache Ant JUnit format.
+`junit` is the legacy Chef InSpec JUnit reporter. It generates an XML report in Apache Ant JUnit format. The output format may be considered nonstandard in several ways. New users are advised to use `junit2`.
 
-'junit' is the legacy Chef InSpec JUnit reporter. It generates nonstandard JUNit XML in several ways. New users are advised to use junit2.
-
-'junit2' is an updated reporter that attempts to match the JUnit "standard", such as it isn't, more closely. It validates against the schema published by [Windy Road](https://github.com/windyroad/JUnit-Schema).
+`junit2` is an updated reporter that provides JUnit output according to the schema published by [Windy Road](https://github.com/windyroad/JUnit-Schema).
 
 ## Implementation Note
 
-This reporter uses the REXML XML generator, but may use more advanced XML systems for testing. This is to keep packaging requirements for Chef InSpec lightweight and free of compiled dependencies.
+This reporter uses the REXML XML generator, but uses the more heavyweight XML library Nokogiri for testing. This is to keep packaging requirements lightweight and free of compiled dependencies.
```

Comment:
```suggestion
This reporter uses the REXML XML generator for implementation, but uses Nokogiri, a more heavyweight XML library, for testing. This design keeps packaging requirements lightweight and free of compiled dependencies.
```

Reply:
@james-stocks "Implementation" may not be the right word choice in this case. Do you have alternative suggestions?

Reply:
Maybe `at runtime`? Or just leave it as `This reporter uses the REXML XML generator, but` ?

Reply:
I went with "runtime," as it's important to differentiate between the two use types and when they're used. We can always come back to edit this later.